tests/issues/issue-1219.php

```diff
@@ -5546,15 +5546,15 @@
             'isFake' => false,
             'isAnonymized' => false,
             'isModified' => false,
-            'CssVersion' => '3',
+            'CssVersion' => '2',
             'Device_Name' => 'Windows Desktop',
             'Device_Maker' => 'unknown',
             'Device_Type' => 'Desktop',
             'Device_Pointing_Method' => 'mouse',
             'Device_Code_Name' => 'Windows Desktop',
             'Device_Brand_Name' => 'unknown',
             'RenderingEngine_Name' => 'Gecko',
-            'RenderingEngine_Version' => '1.6',
+            'RenderingEngine_Version' => 'unknown',
             'RenderingEngine_Maker' => 'Mozilla Foundation',
```

Comment:
Why do you loose the Gecko version information?

Reply:
We don't parse it for this bot.